portfolio/src/main/webapp/script.js

```diff
@@ -25,4 +25,24 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+
+function getdataservlet(){
```

Comment:
If you change this name in index.html to getDataServlet(), then change it here as well